metadata-ingestion/src/datahub/ingestion/api/source_helpers.py

```diff
@@ -325,7 +339,7 @@ def auto_browse_path_v2(
                 yield MetadataChangeProposalWrapper(
                     entityUrn=urn,
                     aspect=BrowsePathsV2Class(
-                        path=_prepend_platform_instance([], platform, platform_instance)
+                        path=prepend_platform_instance([], platform, platform_instance)
```

Comment:
when using a platform instance with manually generated browse path v2, is the source responsible for adding the platform instance or does this helper do it?

I want to make sure we don't have cases where it's prepended twice or more for folders deep in the hierarchy

Reply:
I intended to keep it as responsibility of source to have platform instance prepended in case its manually emitting browse path v2 but I see a corner case now, where the platform instance may get prepended twice - from within helper - to children of the container that emits such browse path v2. 

I can 
1. add a check in `prepend_platform_instance` not to prepend if browse path already has a platform instance entry at its start 
2. or leave the responsibility of prepending platform instance only to source helper - across all sources - i.e. no source should ever prepend platform instance in browse path v2.

I'm inclined to do 2nd for looker but I also think, it wouldn't harm to have a safety net (as mentioned in  point 1). WDYT ?

Reply:
Let’s do the 2nd one

Reply:
Done